catalog-info.yaml

```
@@ -14,8 +14,8 @@ metadata:
       title: "Documentation"
       icon: "Article"
   annotations:
-    openedx.org/arch-interest-groups: "jmbowman"
+    openedx.org/arch-interest-groups: ""
 spec:
-  owner: 
+  owner: "group:frontend-all"
```

```suggestion
  owner: "group:committers-frontend"
```

I changed the team name a few months ago: https://github.com/orgs/openedx/teams/committers-frontend